Cargo.lock

```diff
@@ -6798,7 +6798,7 @@ version = "3.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "da03fa3b94cc19e3ebfc88c4229c49d8f08cdbd1228870a45f0ffdf84988e14b"
 dependencies = [
- "dirs 5.0.1",
+ "dirs 4.0.0",
```

Comment:
Hmm, this downgrade looks a bit weird? `gitui` depends on 5.0.

Reply:
Hmm, I am not sure why cargo decided to resolve to that version when I ran `cargo update git2-hooks`. `shellexpand` has `dirs = { version = ">=4, <6", optional = true }`, and dirs 4 in Cargo.lock comes from `resolve-path`.
I will remove this change.